tools/goctl/pkg/parser/api/parser/api.go

```diff
@@ -31,18 +31,40 @@ func convert2API(a *ast.AST, importManager map[string]placeholder.Type) (*API, e
 	one := a.Stmts[0]
 	syntax, ok := one.(*ast.SyntaxStmt)
 	if !ok {
-		return nil, ast.SyntaxError(one.Pos(), "expected syntax statement, got <%T>", one)
+		syntax = &ast.SyntaxStmt{
+			Syntax: ast.NewTokenNode(token.Token{
+				Type: token.IDENT,
+				Text: token.Syntax,
+			}),
+			Assign: ast.NewTokenNode(token.Token{
+				Type: token.ASSIGN,
+				Text: "=",
+			}),
+			Value: ast.NewTokenNode(token.Token{
+				Type: token.STRING,
+				Text: `"v1"`,
+			}),
+		}
 	}
-	api.Syntax = syntax
 
-	for i := 1; i < len(a.Stmts); i++ {
+	api.Syntax = syntax
+	var hasSyntax, hasInfo bool
+	for i := 0; i < len(a.Stmts); i++ {
```

Comment:
Why change from starting at 1 to starting at 0?

Reply:
Because `syntax` is a required syntax block before, index 0 must be a `syntax` syntax block, so check repeated syntax blocks from index 1. After this pr, `syntax` is optional, so check repeated syntax blocks from index 0